tests/acceptance/administrator/CheckForWarningsAndNoticesCest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package     redCORE
+ * @subpackage  Cept
+ * @copyright   Copyright (C) 2008 - 2019 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+use Step\Acceptance\AbstractStep;
+
+// Load the Step Object Page
+class CheckForWarningsAndNoticesCest
+{
+	/**
+	 * @param AbstractStep $I
+	 * @throws Exception
+	 */
+	public function CheckForWarningsAndNotices(AbstractStep $I)
+	{
+		$I->doAdministratorLogin();
+		$I->Enabletranslations();
+		$I->checkForPhpNoticesOrWarnings('administrator/index.php?option=com_redcore');
```

Comment:
as I think we should move all the urls on Page Class . @NguyenTrang091296  what do you think ?

Reply:
That a great idea. But We should add new page or add on Abstract Page @nhungredweb